docs/narr/cookiecutters.rst

```diff
@@ -3,7 +3,16 @@
 Pyramid cookiecutters
 =====================
 
+.. versionadded:: 1.8
+    Added cookiecutter support.
+
 .. versionadded:: 1.10
+    Merged features from ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` into the single cookiecutter to rule them all, ``pyramid-cookiecutter-starter``.
+
+.. deprecated:: 1.10
+    Merged features from ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` into the single cookiecutter to rule them all, ``pyramid-cookiecutter-starter``.
```

Comment:
Please remove this one sentence in this context, because we're repeating something that user just read. The following two sentences should remain.